packages/systemd-257/org.freedesktop.login1.toml

```diff
@@ -0,0 +1,515 @@
+# Generated programatically, serde/toml doesn't allow to force the "compact"
+# format on serialization:
+# [user.root]
+# rules = []
```

Comment:
For systemd 257, this is missing:

* `ListSessionsEx`
* `Sleep`
* `ReleaseSession`
* `SetTTY`